EOS.code-workspace

```diff
@@ -0,0 +1,29 @@
+{
+  "folders": [
+    {
+      "path": "."
+    }
+  ],
+  "settings": {
+    "[json]": {
+      "editor.defaultFormatter": "esbenp.prettier-vscode"
+    },
+    "[python]": {
+      "editor.defaultFormatter": "charliermarsh.ruff"
+    },
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "editor.formatOnSave": true,
+    "mypy-type-checker.importStrategy": "fromEnvironment",
+    "ruff.importStrategy": "fromEnvironment"
+  },
+  "extensions": {
+    "recommendations": [
+      "ms-python.mypy-type-checker",
+      "ms-python.python",
+      "ms-python.debugpy",
+      "charliermarsh.ruff",
+      "esbenp.prettier-vscode",
+      "ms-python.isort"
+    ]
+  }
+}
```

Comment:
Why should the project have this file at the root level?

Reply:
removed workspace config and moved changes to settings.shared and extensions